include/alpaka/extent/Traits.hpp

```diff
@@ -97,6 +98,8 @@ namespace alpaka
             return getExtents(extent)[Dim<TExtent>::value - 1u];
         else
             return 1;
+
+        ALPAKA_UNREACHABLE({});
```

Comment:
Man, I despise those. I think this is only a problem in older nvcc versions. I hope we can drop those soon!